### ISSUE TITLE
Improved functionality for X and Y

### ISSUE THREAD

**Issue by arq5x** [arq5x/gemini#492]
E.g., de novos on the X in males will be missed as we currently require a HET genotype.

**Comment by arq5x** [comment]
A de novo in PIGA was missed on the X in a male because GATK called mom and dad HOM_REF yet male proband as HOM_ALT (when really HEM_ALT). We should make sure the ``de_novo`` tool uses special logic for this.

**Comment by brentp** [comment]
this was missed by x-linked-denovo?

**Comment by arq5x** [comment]
No, was missed by de_novo. I think we should try to just make ``de_novo`` run the logic in ``x-linked-denovo`` as it is just another chromosome on which to look for de novos. Most people won't think to run both separately.
